prescient/simulator/oracle_manager.py

```diff
@@ -323,17 +323,17 @@ def call_operation_oracle(self, options: Options, time_step: PrescientTime, is_f
                 print("Re-solving SCED after unfixing Quick Start Generators")
                 current_sced_instance = self.engine.enable_quickstart_and_solve(current_sced_instance, options)
 
-        self.simulator.plugin_manager.invoke_after_operations_callbacks(options, self.simulator, current_sced_instance)
-
-
```

Comment:
I don't remember for sure, but I thought we put this *before* the LMPs for a reason, such as allowing a plugin to modify the model here before LMPs are solved.  Is this where Xian needed to adjust generator output?  I just want to make sure this wasn't overlooked.

Reply:
My understanding is that adjusting the generator output happens in the callback after this, and that Xian may need to track the dispatch of the generator here. I had the same nagging feeling you do about this, but it's difficult for me to find a justification.

Adjusting the generator output in the LMP SCED would not do anything as-is -- the LMP SCED (despite its name), also decides dispatches as a part of its solution process. Fixing the dispatch from some generator would have the effect of "pricing it out," i.e., it wouldn't contribute to setting the LMP.

In fact, in some settings it may be appropriate to solve a single problem instead of a SCED+LMP-SCED. Moving this callback would facilitate such a change.

Reply:
@xiangao1, Do you have any insights here?

Reply:
@darrylmelander @bknueven I am sorry about the late reply! I must have missed the Github notification. Currently, my plan for tracking the dispatch for a generator, which may end up adjusting the actual dispatch for that generator, is after the SCED solve and with LMP available. The point in green is what I mean:
![image](https://user-images.githubusercontent.com/39804296/95596054-e6a60700-0a1a-11eb-98f7-d9505b00a4b6.png)

So I won't use the plugin right after the SCED solve at this moment. 

Thanks!

Reply:
Thanks @xiangao1! If anything, it seems like this plugin *should* be moved to the location suggested by this PR, and perhaps the `lmp_sced` should also be passed into it. The later can be a separate issue/PR.